CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -157,11 +163,11 @@ local function OnHotbarEquipPrimary(actionName, state, obj)
 	for tool, slot in pairs(Tools) do
 		if slot.hovered then
 			slot.OnClick()
+			return
```

Comment:
We can if you want. My next issue calls for quite a bit of changes to this script so I was going to leave it for then.